apps/client/src/components/qna-posting/QnaBox/index.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+import { Button, Flex, Stack, Text } from '@sickgyun/ui';
+import { useState } from 'react';
+import { QNA_CATEGORY } from '@/constants/qna-category';
+
+const QnaBox = () => {
+  const [activeCategory, setActiveCategory] = useState(0);
```

Comment:
```suggestion
  const [activeCategoryIndex, setActiveCategoryIndex] = useState(0);
```

Index를 붙히면 숫자인것이 명시적으로 들어날수있을것같아요

Reply:
그렇군요 알겠습니다.